sql/testdata/datetime

```diff
@@ -85,6 +85,10 @@ true true
 
 # TIMESTAMP/DATE builtins.
 
+# SET TIME ZONE affects the builtins
+statement ok
+SET TIME ZONE INTERVAL '-7h'
```

Comment:
If adding this statement doesn't change the outcomes of any of the tests below, it doesn't really prove anything. We need to show that running the same queries in two different time zones gives different results.